### PR TITLE
Nepali 🇳🇵 translation for cheat sheet added

### PR DESCRIPTION
**Pull request by nirajacharyaa**
## Overview
 - Added a Nepali translation for the latest version of the [English cheatsheet](https://github.com/github/training-kit/blob/master/downloads/github-git-cheat-sheet.md)
 - Added a link to the [index.html](https://github.com/github/training-kit/blob/master/index.html)

### Review
I request @ujjwalnp @suryachandthakuri to review the PR

### CODE REVIEW

**Review by ujjwalnp — APPROVED** [review]
### Review for #921 by @ujjwalnp

- Nepali translatation provided is quite accurate.
- Necessary changes have also been taken.

File: downloads/ne/github-git-cheat-sheet.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+title: GitHub Git Cheat Sheet
+byline: Git is the open source distributed version control system that facilitates GitHub activities on your laptop or desktop. This cheat sheet summarizes commonly used Git command line instructions for quick reference.
+leadingpath: ../../
+---
+
+{% capture colOne %}
+
+## इन्स्टल गर्नुहोस
+
+### Github डेस्कटप
```
Comment (parkerbxyz) [inline downloads/ne/github-git-cheat-sheet.md:13]:
```suggestion
### GitHub डेस्कटप
```

File: downloads/ne/github-git-cheat-sheet.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+यसले तपाइले commit गर्दा संलग्न गर्न चाहानु भएको नाम सेट गर्दछ
+
+`$ git config --global user.email "[email address]"`
+
+यसले तपाइले commit गर्दा संलग्न गर्न चाहानु भएको इमेल सेट गर्दछ
+
+`$ git config --global color.ui auto`
+
+यसले कमाण्ड लाइन आउटपुटको रङकरण गर्दछ
```
Comment (parkerbxyz) [inline downloads/ne/github-git-cheat-sheet.md:35]:
```suggestion
यसले कमाण्ड लाइन आउटपुट रङकरण गर्छ
```

File: downloads/ne/github-git-cheat-sheet.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+यसले नया branch बनाउनेछ
+
+`$ git switch -c [branch-name]`
+
+यसले तपाइको working directory लाई तपाइले चयन गर्नुभएको branch मा परिवर्तन गर्नेछ
+
+`$ git merge [branch]`
+
+यसले तपाइले चयन गर्नुभएको branch को history ल ाई वर्तमान branch मा मिस्रित गराउनेछ l यो प्रायजसो pull request हरुमा गरिन्छ, तर यो पनि एउटा महत्वपुर्ण कार्य हो l
```
Comment (parkerbxyz) [inline downloads/ne/github-git-cheat-sheet.md:51]:
```suggestion
यसले तपाइले चयन गर्नुभएको branch को history ल ाई वर्तमान branch मा मिस्रित गराउनेछ।  यो प्रायजसो pull request हरुमा गरिन्छ, तर यो पनि एउटा महत्वपुर्ण कार्य हो।
```

File: downloads/ne/github-git-cheat-sheet.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+यसले तपाइले commit गर्दा संलग्न गर्न चाहानु भएको इमेल सेट गर्दछ
+
+`$ git config --global color.ui auto`
+
+यसले कमाण्ड लाइन आउटपुटको रङकरण गर्दछ
+
+## Branches
+
+Branche हरु Git संग काम गर्नलाई महत्त्वपूर्ण हुन्छन् l अहिले "checkdout" branch मा तपाइले गरेको commit हरु राखिनेछ l Branch हेर्नकालागि `git status` को प्रयोग गर्न सक्नुहुनेछ l
```
Comment (parkerbxyz) [inline downloads/ne/github-git-cheat-sheet.md:39]:
```suggestion
Branches हरु Git संग काम गर्नलाई महत्त्वपूर्ण हुन्छन्।  अहिले "checked out" branch मा तपाइले गरेको commit हरु राखिनेछ।  Branch हेर्नकालागि `git status` को प्रयोग गर्न सक्नुहुनेछ।
```

File: downloads/ne/github-git-cheat-sheet.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+यसले तपाइको working directory लाई तपाइले चयन गर्नुभएको branch मा परिवर्तन गर्नेछ
+
+`$ git merge [branch]`
+
+यसले तपाइले चयन गर्नुभएको branch को history ल ाई वर्तमान branch मा मिस्रित गराउनेछ l यो प्रायजसो pull request हरुमा गरिन्छ, तर यो पनि एउटा महत्वपुर्ण कार्य हो l
+
+`$ git branch -d [branch-name]`
+
+यसले तपाइले चयन गर्नुभएको branch मेटाउनेछ
```
Comment (parkerbxyz) [inline downloads/ne/github-git-cheat-sheet.md:55]:
```suggestion
यसले तपाइले चयन गर्नुभएको branch मेटाउने
```

File: downloads/ne/github-git-cheat-sheet.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+
+<div class="col-md-6">
+{{ colOne | markdownify }}
+</div>
+
+{% capture colTwo %}
+
+## Repository बनाउनुहोस
+
+नया repository आफ्नो कम्प्युटरमा बनाउन सकिन्छ अथवा कसैले बनाएको repository क्लोन गर्न सकिन्छ l यदी तपाइले आफ्नो कम्प्युटरमा repository बनाउनु भयो भने उक्त repository लाई पछि Github मा push गर्नुपर्ने हुन्छ l
```
Comment (parkerbxyz) [inline downloads/ne/github-git-cheat-sheet.md:67]:
```suggestion
नया repository आफ्नो कम्प्युटरमा बनाउन सकिन्छ अथवा कसैले बनाएको repository क्लोन गर्न सकिन्छ।  यदी तपाइले आफ्नो कम्प्युटरमा repository बनाउनु भयो भने उक्त repository लाई पछि GitHub मा push गर्नुपर्ने हुन्छ।
```

File: downloads/ne/github-git-cheat-sheet.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+
+`$ git init`
+
+यदी तपाइले काम गर्दै गरेको directory मा git init कमाण्ड रन गर्नुभयो भने उक्त directory लाई Git repository मा परिणत गर्नेछ l `git init` कमाण्ड प्रयोग गरेपछि, उक्त directory लाई खाली Git repository संग लिंक गर्न निम्न कमाण्ड प्रयोग गर्नुहोस् :
```
Comment (parkerbxyz) [inline downloads/ne/github-git-cheat-sheet.md:71]:
```suggestion
यदी तपाइले काम गर्दै गरेको directory मा `git init` कमाण्ड रन गर्नुभयो भने उक्त directory लाई Git repository मा परिणत गर्नेछ। `git init` कमाण्ड प्रयोग गरेपछि, उक्त directory लाई खाली Git repository संग लिंक गर्न निम्न कमाण्ड प्रयोग गर्नुहोस् :
```

File: downloads/ne/github-git-cheat-sheet.md [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+
+`$ git remote add origin [url]`
+
+यसले तपाइको कम्प्युटर मा भएको लोकल repository लाई remote repository संग लिंक गर्नेछ l url ले Github मा भएको repository लाई औंलााउनेछ l
```
Comment (parkerbxyz) [inline downloads/ne/github-git-cheat-sheet.md:75]:
```suggestion
यसले तपाइको कम्प्युटर मा भएको लोकल repository लाई remote repository संग लिंक गर्नेछ। URL ले GitHub मा भएको repository लाई औंलााउनेछ।
```

File: downloads/ne/github-git-cheat-sheet.md [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+
+`$ git clone [url]`
+
+यसले url को, repository लाई डाउनलोड अथवा क्लोन गर्नेछ l क्लोनमा सबै फाइल, branch अनि commit हरु संलग्न गरिनेछ l
```
Comment (parkerbxyz) [inline downloads/ne/github-git-cheat-sheet.md:79]:
```suggestion
यसले URL को, repository लाई डाउनलोड अथवा क्लोन गर्नेछ। क्लोनमा सबै फाइल, branch अनि commit हरु संलग्न गरिनेछ।
```

File: downloads/ne/github-git-cheat-sheet.md [hunk 10]
```diff
@@ -0,0 +1,183 @@
...
+
+## .gitignore फाइल
+
+कहिले कहिँ कुनै फाइलहरु Git ले ट्रयाक न गर्नु राम्रो विचार हुन सक्छ l यो प्रक्रिया एउटा फाइल मा गरिन्छ जसको नाम `.gitignore` हो l यसका लागि तपाइले टेम्पलेट [github.com/github/gitignore](https://github.com/github/gitignore) मा भेटाउन सक्नुहुनेछ l
```
Comment (parkerbxyz) [inline downloads/ne/github-git-cheat-sheet.md:83]:
```suggestion
कहिले कहिँ कुनै फाइलहरु Git ले ट्रयाक न गर्नु राम्रो विचार हुन सक्छ। यो प्रक्रिया एउटा फाइल मा गरिन्छ जसको नाम `.gitignore` हो। यसका लागि तपाइले टेम्पलेट [github.com/github/gitignore](https://github.com/github/gitignore) मा भेटाउन सक्नुहुनेछ।
```

**Review by parkerbxyz — COMMENTED** [review]
Thanks for taking the time to translate this document and open this pull request, @nirajacharyaa! ✨ I left several suggested changes, most of which where suggested by Google Translate. Feel free to reject any of the suggestions, as they may not be good ones.

**Comment by nirajacharyaa** [comment]
@parkerbxyz Thank you for reviewing those mistakes, applied your suggestions. Nevertheless, the translation is now good, there may be some barely noticeable mistakes. It's a pleasure to contribute to localization. 💫

**Review by parkerbxyz — APPROVED** [review]

**Pull request merged**